test/local_creds_test.go

```diff
@@ -0,0 +1,217 @@
+/*
+ *
+ * Copyright 2020 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package test
+
+import (
+	"context"
+	"fmt"
+	"net"
+	"strings"
+	"testing"
+	"time"
+
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/credentials"
+	"google.golang.org/grpc/credentials/local"
+	"google.golang.org/grpc/peer"
+	"google.golang.org/grpc/status"
+
+	testpb "google.golang.org/grpc/test/grpc_testing"
+)
+
+func testE2ESucceed(network, address string) error {
```

Comment:
I know this is in a file called `local_creds_test.go`, but it's in a huge testing package.  Please give this a name that includes `LocalCreds`.  Same for everything global in this file, especially the `Test____` functions.

Reply:
Done.